### ISSUE TITLE
Install fails if webpack.mix.js does not exist

### ISSUE THREAD

**Issue by than** [edalzell/statamic-tailwind#2]
```
~/Sites/thanland master* ⇡
❯ pp tailwind:install


  [League\Flysystem\FileNotFoundException]
  File not found at path: site/themes/thanland/webpack.mix.js

```

**Comment by edalzell** [comment]
That file is included in the addon, where is that error coming from?

**Comment by edalzell** [comment]
Fixed in 1.0 release
